app/views/admin/user_trainings/edit.haml

```diff
@@ -21,9 +21,10 @@
       %tr
         %th Course
         %th Progress
+        %th Renewal Required After
     %tbody
       - @user.required_training_courses.each do |course|
         %tr
           %td= course.name_with_subdomain
           %td= @user.training_status(course)
-    
\ No newline at end of file
+          %td= @user.training_renewal_date(course)
```

Comment:
Maybe this should say `not required` or `never` if the date is blank?

Reply:
I'll update the user concern method to return 'Never' when the `months_to_expiration` field is nil. This should allow it to still be blank when there is an expiration but the training hasn't been completed.